app/src/main/java/com/sopt/gongbaek/presentation/type/GroupInfoChipType.kt

```diff
@@ -0,0 +1,97 @@
+package com.sopt.gongbaek.presentation.type
+
+import androidx.annotation.StringRes
+import androidx.compose.ui.graphics.Color
+import com.sopt.gongbaek.R
+import com.sopt.gongbaek.ui.theme.gray01
+import com.sopt.gongbaek.ui.theme.gray02
+import com.sopt.gongbaek.ui.theme.gray06
+import com.sopt.gongbaek.ui.theme.gray07
+import com.sopt.gongbaek.ui.theme.gray08
+import com.sopt.gongbaek.ui.theme.gray09
+import com.sopt.gongbaek.ui.theme.mainOrange
+import com.sopt.gongbaek.ui.theme.subBlue
+import com.sopt.gongbaek.ui.theme.subGreen
+import com.sopt.gongbaek.ui.theme.subOrange
+import com.sopt.gongbaek.ui.theme.white
+
+enum class GroupInfoChipType(
+    @StringRes val label: Int,
+    val backgroundColor: Color = subOrange,
+    val fontColor: Color = mainOrange
```

Comment:
[1]: 직접 접근하는 방식 말고, 정의된 defaultGongBaekColors를 통해 접근해주세욤 👍🏻
```suggestion
    val backgroundColor: Color = defaultGongBaekColors.subOrange,
    val fontColor: Color = defaultGongBaekColors.mainOrange
```

Reply:
GongBaekTheme. ~ 으로 접근하는 방식은 어떨까요!

Reply:
`    val backgroundColor: Color = GongBaekTheme.colors.subOrange,
`
이런 식으로 접근하게 되면 GongBaekTheme.colors.subOrange가 composable 함수에서만 접근 가능한 값이라 오류가 발생합니다!
val backgroundColor: Color = defaultGongBaekColors.subOrange,
이렇게 수정해볼게요 !